packages/admin-ui/ui/src/domain/orders/details/allocations/edit-allocation-modal.tsx

```diff
@@ -1,47 +1,72 @@
-import { AllocationLineItemForm } from "./allocate-items-modal"
 import { Controller, useForm, useWatch } from "react-hook-form"
```

Comment:
todo: File name should reflect component i.e. `edit-allocation-drawer.tsx`

Reply:
yes, Since this will be a lot of files it will be a separate pr migrating everything to be named "Reservation" rather than "allocation" (core-1380)